src/components/Button/Button.js

```diff
@@ -23,48 +23,47 @@ const propTypes = {
   onClick: PropTypes.func.isRequired,
 };
 
+const NOOP = () => {};
+
 const Button = props => {
   const [isElementDisabled, customOverrides] = useSelector(state => [
     selectors.isElementDisabled(state, props.dataElement),
     selectors.getCustomElementOverrides(state, props.dataElement),
   ]);
 
-  props = { ...props, ...customOverrides };
   const {
+    disable,
```

Comment:
back when i worked with Angular 3rd party libraries. some values of "undefined" = true. what is the plan for this?

Reply:
I'm not sure what you mean by `some values of "undefined" = true`. Do you have any examples that I can look at?

Reply:
looking around it seems I can't seem to remember the examples I found in the past.

From thinking around, undefined usually means false.